src/components/projects/projectId/attributes/attributeId/LLM/llmTemplates.ts

```diff
@@ -0,0 +1,171 @@
+
+export const TEMPLATE_EXAMPLES = {
+  REASONED_CLICKBAIT: {
+    templatePrompt: "You are running your own information network and need to ensure no clickbait news articles are published. To ensure your answer can be validated always provide a reason and your final result being either 'yes' or 'no' to the question is this clickbait. JSON Schema {\"reason\": <reasoning>, \"result\":<result>}",
+    questionPrompt: "News article: '{{headline}}'"
+  },
+  SHORT_SUMMARY_REFERENCE: {
+    templatePrompt: "Summarize the given text into two short sentences. Don't add any new keys only provide {\"result\":\"<summar>\"}",
+    questionPrompt: "{{reference}}"
+  },
+  META_DATA_EXTRACTION_REFERENCE: {
+    templatePrompt: `Your task is to extract meta data from the given text following this structure: 
+{
+  "linguistic_metadata": {
+    "language": "",  // ISO 639-1 language code
+    "reading_complexity": {
+      "vocabulary_level": "",  // Possible values: "basic", "moderate", "advanced"
+      "sentence_structure": "",  // Possible values: "simple", "moderate", "complex"
+      "readability_level": "" //reading level "pre school", "high school", "college"
+    },
+    "grammar_and_style": ""  // Possible values: "formal", "informal", "technical", "creative"
+  },
+  "structural_metadata": {
+    "document_structure": {
+      "contains_headings": , // true or false
+      "contains_subcategories": ,  // true or false
+      "format": ""  // Possible values: "paragraph", "list", "table"
+    }
+  },
+  "semantic_metadata": {
+    "key_themes": [], //list all key themes - max 10
+    "keywords": [] //list all keywords - max 10
+  },
+  "sentiment_metadata": {
+    "overall_sentiment": "",  // Possible values: "positive", "neutral", "negative"
+    "emotional_tone": [] //list all emotional tones - Possible values: "detachment", "caution", "optimism", "warning", "encouragement"
+  },
+  "repeated_phrases": [] //list repeated phrases if any,
+  "contextual_metadata": {
+    "target_audience": [], //make an educated guess on who would be the target audience - max 3
+    "purpose": "" //make an educated guess on what the purpose of the text is - max 2 sentences
+  },
+  "sentence_types": [] // list all sentence types Possible values: "explanatory", "descriptive", "narrative", "directive"
+}
+
+Don't remove any keys from the structure even if they are empty.`,
+    questionPrompt: "{{reference}}"
+    // removed since llms cant count words,
```

Comment:
comments (not sure if intentionally wanted)

Reply:
intentional so the next one doesn't fall for the same trap as i did :)